components/engine/Cargo.toml

```diff
@@ -18,12 +18,12 @@ lazy_static = "1.3"
 prost = { version = "0.5", optional = true }
 protobuf = "2.8"
 quick-error = "1.2.2"
-raft = { version = "0.6.0-alpha" , default-features = false }
+raft = { version = "0.6.0-alpha", default-features = false }
 serde = "1.0"
 serde_derive = "1.0"
 slog = { version = "2.3", features = ["max_level_trace", "release_max_level_debug"] }
 slog-global = { version = "0.1", git = "https://github.com/breeswish/slog-global.git", rev = "0e23a5baff302a9d7bccd85f8f31e43339c2f2c1" }
-sys-info = "0.5.7"
+sysinfo = { git = "https://github.com/tikv/sysinfo.git", branch = "tikv"}
```

Comment:
Why another fork?

Reply:
The official repo doesn't contain some useful features (e.g retrieve network interface information/ `sysctl` list etc). And merge the extra useful interfaces to the upstream is in the plan, but not current because the extra interfaces which we added don't support Windows.

Reply:
Maybe we will add Windows support after completing https://github.com/pingcap/tidb/issues/13567.